hcl/hcl.go

```diff
@@ -1154,13 +1202,40 @@ func (p *TerramateParser) parseTerramateSchema() (Config, error) {
 		}
 	}
 
+	errs.Append(validateRunEnv(tmconfig))
+
 	if err := errs.AsError(); err != nil {
 		return Config{}, err
 	}
 
 	return tmconfig, nil
 }
 
+func validateRunEnv(config Config) error {
+	if config.Terramate == nil ||
+		config.Terramate.Config == nil ||
+		config.Terramate.Config.Run == nil ||
+		config.Terramate.Config.Run.Env == nil {
+		return nil
+	}
+
+	errs := errors.L()
+	attrs := map[string]Attribute{}
+
+	for _, attr := range config.Terramate.Config.Run.Env.Attributes {
+		name := attr.Value().Name
+		if _, ok := attrs[name]; ok {
```

Comment:
isn't this ensured by the hcl language spec and also validated by hcl lib?

got this in the playground:
```
playground:13,4-5: Attribute redefined; The argument "a" was already set at playground:12,4-5. Each argument may be set only once.
```

Reply:
Not ensured if they come from different blocks or different files. Like this is allowed:

```
block {
  a = 666
}

block {
  a = 666
}
```

Here we detect conflicts accross multiple blocks/files, so I think we need this (you can check the tests for these errors too).

Reply:
gotcha